### PR TITLE
Fix fleet tab pilot lists not rendering

### LINKED ISSUE #93: BUG: Fix hired pilots not loading

### PR DESCRIPTION
**Pull request by maxwellpark**

### CODE REVIEW

File: Assets/Scripts/DataModels/Pilots/PilotAccessors.cs [hunk 1]
```diff
@@ -35,10 +35,6 @@ public bool IsAssignedToShip
     {
         get => saveData.isAssignedToShip; set { saveData.isAssignedToShip = value; }
     }
-
-    public Ship Ship 
-    {
-        get => ShipsManager.Instance.Ships.FirstOrDefault(s => s.Pilot == this);
```
Comment (maxwellpark) [inline Assets/Scripts/DataModels/Pilots/PilotAccessors.cs:41]: Scrapped this and went back to the old, circular reference. Don't have to look up the ship each time, and don't need one-to-one between ships and pilots (which atm is annoying lol, and throwing null ref).

File: Assets/Scripts/Singletons/DataModelSingletons/PilotsManager.cs [hunk 2]
```diff
@@ -69,12 +69,12 @@ public void HirePilot(Pilot pilot)
...
     public Pilot[] GetHiredPilots()
     {
-        return Instance.Pilots.Where(p => p.IsHired) as Pilot[];
+        return Instance.Pilots.Where(p => p.IsHired).ToArray();
     }
 
     public Pilot[] GetPilotsForHire()
     {
-        return Instance.Pilots.Where(p => !p.IsHired) as Pilot[];
+        return Instance.Pilots.Where(p => !p.IsHired).ToArray();
```
Comment (ambid17) [inline Assets/Scripts/Singletons/DataModelSingletons/PilotsManager.cs:77]: cant believe i didn't catch this the first time around lmao

File: Assets/Scripts/UI/Terminal/PilotsUI/PilotsUI.cs [hunk 3]
```diff
@@ -92,8 +92,16 @@ private void CleanScrollView(Transform scrollViewContent)
 	private void ShowPilotProfilePanel(Pilot pilot)
 	{
 		SetPilotListVisibility(visible: false);
-		shipAvatar.sprite = pilot.Ship.Avatar;
-		pilotAvatar.sprite = pilot.Avatar;
+
+		if (pilot.Ship != null && pilot.Ship.Avatar != null)
```
Comment (ambid17) [inline Assets/Scripts/UI/Terminal/PilotsUI/PilotsUI.cs]: add debug statements in else

**Review by ambid17 — APPROVED** [review]

File: Assets/Scripts/Singletons/DataModelSingletons/PilotsManager.cs (hunk 2 above)
Comment (maxwellpark) [reply Assets/Scripts/Singletons/DataModelSingletons/PilotsManager.cs:77]: i dont remember doing this lol
i think i had it as toarray originally as it was working, then did this for some reason.

**Pull request merged**